config/test.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
+};
```

Comment:
Yup makes sense. Thanks! I realized this and made the change but didn't actually commit/push it earlier.